### PR TITLE
Add `export` subcommand to `nydus-image`

### PR DESCRIPTION
**Pull request by jiangliu**
Add `export` subcommand to `nydus-image`, which exports RAFS filesystems as raw block device images or tar files.
It will be enhanced to generate integrity data for `dm-verity` and/or `fs-verity`.

### CODE REVIEW

File: src/bin/nydus-image/main.rs [hunk 1]
```diff
@@ -1443,4 +1507,63 @@ impl Command {
...
+                        "type": "filecache",
+                        "filecache": {
+                            "work_dir": "LOCAL_FS_DIR"
+                        }
+                    },
+                    "metadata_path": "META_FILE_PATH"
+                }
+            }"#;
+            let config = config
+                .replace("LOCAL_FS_DIR", dir)
```
Comment (imeoer) [inline src/bin/nydus-image/main.rs:1540]: Maybe use `format!` here?

**Comment by imeoer** [comment]
Can we give a brief doc about why & how to use `export` command in `docs/nydus-image.md` ?

File: src/bin/nydus-image/main.rs (hunk 1 above)
Comment (jiangliu) [reply src/bin/nydus-image/main.rs:1540]: fixed

**Comment by jiangliu** [comment]
> Can we give a brief doc about why & how to use `export` command in `docs/nydus-image.md` ?

I will submit a dedicated PR to update the doc about `nydus-image export`.

**Review by imeoer — APPROVED** [review]

**Pull request merged**